src/Provider/AbstractProvider.php

```diff
@@ -217,12 +227,8 @@ public function getAccessToken($grant = 'authorization_code', $params = [])
 
                 break;
         }
-
-        if (isset($result['error']) && ! empty($result['error'])) {
-            // @codeCoverageIgnoreStart
-            throw new IDPException($result);
-            // @codeCoverageIgnoreEnd
-        }
+        $checker = $this->errorChecker;
+        $checker($result);
```

Comment:
I guess this means that errorChecker must always be a callable? We should probably do a check for that somewhere (the constructor?) and throw an exception if it's not the case. In v1.0, as I said on your other pull request, I'd like to change these properties to protected and have real getters/setters on them, so that we can type-hint for things like `callable`.